ghost/i18n/locales/es/portal.json

```diff
@@ -9,168 +9,168 @@
+    "Try free for {{amount}} days, then {{originalPrice}}.": "Prueba gratis por {{amount}} dias, luego de dicho plazo deberás pagar {{originalPrice}}.",
     "Unlock access to all newsletters by becoming a paid subscriber.": "Desbloquea el acceso a todos los boletines convirtiéndote en un suscriptor pago.",
-    "Unsubscribe from all emails": "Cancelar suscripción a todos los correos electrónicos",
+    "Unsubscribe from all emails": "Cancelar suscripción de todos los correos electrónicos",
     "Unsubscribed": "Dado de baja",
-    "Unsubscribed from all emails.": "",
+    "Unsubscribed from all emails.": "De suscribete de todos los correos electrónicos",
     "Unsubscribing from emails will not cancel your paid subscription to {{title}}": "Cancelar la suscripción a los correos electrónicos no cancelará tu suscripción de pago a {{title}}",
     "Update": "Actualizar",
     "Update your preferences": "Actualiza tus preferencias",
     "Verification link sent, check your inbox": "Se ha enviado un enlace de verificación, revisa tu bandeja de entrada",
-    "Verify your email address is correct": "Verifica que tu dirección de correo electrónico sea correcta",
+    "Verify your email address is correct": "Verifica que tu dirección de correo electrónico para que funcione correctamente"
     "View plans": "Ver planes",
     "We couldn't unsubscribe you as the email address was not found. Please contact the site owner.": "No pudimos cancelar tu suscripción ya que no se encontró la dirección de correo electrónico. Por favor, contacta al propietario del sitio.",
     "Welcome back, {{name}}!": "¡Bienvenido de nuevo, {{name}}!",
     "Welcome back!": "¡Bienvenido de nuevo!",
     "Welcome to {{siteTitle}}": "Bienvenido a {{siteTitle}}",
-    "When an inbox fails to accept an email it is commonly called a bounce. In many cases, this can be temporary. However, in some cases, a bounced email can be returned as a permanent failure when an email address is invalid or non-existent.": "Cuando una bandeja de entrada no acepta un correo electrónico, comúnmente se le llama rebote. En muchos casos, esto puede ser temporal. Sin embargo, en algunos casos, un correo electrónico rebotado puede devolverse como una falla permanente cuando una dirección de correo electrónico no es válida o no existe.",
+    "When an inbox fails to accept an email it is commonly called a bounce. In many cases, this can be temporary. However, in some cases, a bounced email can be returned as a permanent failure when an email address is invalid or non-existent.": "Cuando una bandeja de entrada no acepta un correo electrónico, se le llama rebote de correo electrónico. En muchos casos, esto puede ser temporal, dado que el correo puede ser añadido a lista blanca. Sin embargo, en algunos casos, un correo electrónico rebota por una falla permanente, dado que dicha dirección no es válida o no existe.",
```

Comment:
Whitelisting is a separate concept that is not in the original text.  Perhaps remove that phrase?

Reply:
Got it, deleted it.